docs/administrator_guide/enrichment/data_transforms.md

```diff
@@ -206,6 +207,17 @@ When `AI Extracted Text Cleansing` is performed on a document, `Trace AI Extract
   1. Cleansing is meant to only work on email documents. This means emails that have clear and defined headers will get cleansed. Email headers need to contain From, Sent, Subject, Body, and at least one of To, CC, or BCC. If these headers can't be parsed, then cleansing will fail with a status of Warning - Document Error. No other type of documents are currently supported for cleansing.
   2. AI Extracted Text Cleansing transformation occurs before any Replace transformations take place. This means, if there are Replace transform that target non-authored content, they will not take effect if that portion of the text is removed by the AI Extracted Text Cleansing transform first.
 
+### Trace Conversation Thread Data Transformation
```

Comment:
@daccetta2  I don't believe that this is a brand new Transform. I believe it is just a new field added to the existing "Email Thread Deduplication" setting with the AI Text Cleansing transformation. I've update the documentation to reflect that, but can you reach out to Konrad and confirm.